lxd/db_update.go

```diff
@@ -15,6 +15,16 @@ import (
 	log "gopkg.in/inconshreveable/log15.v2"
 )
 
+func dbUpdateFromV25(db *sql.DB) error {
+	if err := dbProfileCreateDocker(db); err != nil {
```

Comment:
Hmm, that needs to be done using pure SQL here, not calling an helper function.

Using the function is certainly more convenient but if we ever add another column to any of the affected tables, then this function call will fail and prevent daemon startup.
It will fail because the sql in the function will be inserting to or reading from a DB column which doesn't exist at this point in the schema update process (introduced in a later schema update).

We ran into this very problem recently and that caused some DB corruption for folks, so please re-implement using pure SQL here.